etl-driver/src/main/java/com/wipro/ats/bdre/im/etl/api/CreateRawBaseTables.java

```diff
@@ -283,15 +279,16 @@ private void checkAndCreateRawTable(String dbName, String tableName, String ddl)
         try {
             LOGGER.debug("Reading Hive Connection details from Properties File");
             Connection con = getHiveJDBCConnection(dbName);
-            Statement stmt = con.createStatement();
-            ResultSet rs = stmt.executeQuery("SHOW TABLES LIKE '" + tableName + "'");
+
+            PreparedStatement pst = con.prepareStatement("SHOW TABLES LIKE '" + tableName + "'");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "SHOW TABLES LIKE '" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) "tableName" is provided externally to the method and not sanitized before use. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS2077)